corehq/messaging/scheduling/models/content.py

```diff
@@ -162,26 +192,38 @@ def send(self, recipient, logged_event, phone_entry=None):
             logged_subevent.error(e.error_type, additional_error_text=e.additional_text)
             return
 
-        if is_trial and EmailUsage.get_total_count(logged_event.domain) >= self.TRIAL_MAX_EMAILS:
+        if is_trial and EmailUsage.get_total_count(domain) >= self.TRIAL_MAX_EMAILS:
             logged_subevent.error(MessagingEvent.ERROR_TRIAL_EMAIL_LIMIT_REACHED)
             return
 
-        metrics_counter('commcare.messaging.email.sent', tags={'domain': logged_event.domain})
-        send_mail_async.delay(subject, message,
-                              [email_address],
-                              messaging_event_id=logged_subevent.id,
-                              domain=logged_event.domain,
-                              use_domain_gateway=True)
+        metrics_counter('commcare.messaging.email.sent', tags={'domain': domain})
+        if toggles.RICH_TEXT_EMAILS.enabled(domain) and html_message:
+            send_html_email_async.delay(
```

Comment:
we'll also want to pass `use_domain_gateway=True`

Reply:
https://github.com/dimagi/commcare-hq/pull/33798/commits/8b1cbfaf1b3b14434c0e590a4ecfda977748ce1d